docker/airflow/local_airflow.md

```diff
@@ -148,6 +148,8 @@ Here is how you can debug them.
 In this case, clearly the connection `datahub-rest` has not been registered. Looks like we forgot to register the connection with Airflow!
 Let's execute Step 4 to register the datahub connection with Airflow. 
 
+In case you registered the connection but are still seeing `Failed to establish a new connection` check if the connection is `http://datahub-gms:8080` and not `http://localhost:8080` 

```

Comment:
nit: suggest rewriting as:

In case the connection was registered successfully but you are still seeing `Failed to establish a new connection`, check to ensure that the connection endpoint is `http://datahub-gms:8080` and not `http://localhost:8080`.

Reply:
Made changes